src/index.css

```diff
@@ -1,13 +1,186 @@
-:root {
+/* ------------ Font Imports ------------ */
```

Comment:
I haven't used Tailwind yet so this was nice to see to learn! Great with clear comments